docs/api/web-frame.md

```diff
@@ -12,9 +16,9 @@ const {webFrame} = require('electron')
 webFrame.setZoomFactor(2)
 ```
 
-## Methods
```

Comment:
This is why linting has failing, if you revert this back to `Methods` everything should work.  `webFrame` is not a class, it's a module (that happens to be an the instance of a native `WebFrame` class) but it's a module 😄

Reply:
I'll revert it to fix the problem, but webFrame is not a module, unless Electron has its own concept of modules.

- it is an exported object of the electron module `require('electron').webFrame`
- it is an instance of the WebFrame class (which is not exported because you can't create new frames)
- it represents the top frame of the current window
- it's methods are instance methods of the WebFrame class
- till now it was a singleton (the only WebFrame instance you could obtain) but with this pull request you can get others

Reply:
Went back to Methods, still failing: Failed to lint electron.d.ts